classes/Cart.php

```diff
@@ -4362,6 +4362,10 @@ public function getProductCustomization($id_product, $type = null, $not_in_cart
             return [];
         }
 
+        if (0 === (int) $this->id) {
```

Comment:
Personally, I would comment it 😄Something like:
```
// Do not return anything, if there is no cart. Otherwise, all customers with disabled cookies would get the same customization.
```

Reply:
@Prestaworks I see that you added a comment, but other Daniel's one is a bit more clear to me, as this issue is strictly related to the cookies

Reply:
I guess that even if you accept cookies, if there is no cart set you could get other users data also. id_cart = 0 in the sql query.